torch/distributed/__init__.py

```diff
@@ -385,25 +382,22 @@ def all_gather_multigpu(output_tensor_lists,
         output_tensor_lists (List[List[Tensor]]): Output lists. It should
             contain correctly-sized tensors on each GPU to be used for output of
             the collective.
-
-            e.g. output_tensor_lists[i] contains the all_gather
-            result that resides on the GPU of input_tensor_list[i].
-
-            Note that each element of output_tensor_lists[i] has the size of
-            world_size * len(input_tensor_list), since the function all gathers
-            the result from every single GPU in the group. To interpret each
-            element of output_tensor_list[i], note that input_tensor_list[j] of
-            rank k will be appear in
-            output_tensor_list[i][rank * world_size + j]
-
-            Also note that len(output_tensor_lists), and the size of each
-            element in output_tensor_lists (each element is a list,
-            therefore len(output_tensor_lists[i])), need to be the same
+            e.g. ``output_tensor_lists[i]`` contains the all_gather
+            result that resides on the GPU of ``input_tensor_list[i]``.
+            Note that each element of ``output_tensor_lists[i]`` has the size of
+            ``world_size * len(input_tensor_list)``, since the function all
+            gathers the result from every single GPU in the group. To interpret
+            each element of ``output_tensor_list[i]``, note that
+            ``input_tensor_list[j]`` of rank k will be appear in
+            ``output_tensor_list[i][rank * world_size + j]``
+            Also note that ``len(output_tensor_lists)``, and the size of each
+            element in ``output_tensor_lists`` (each element is a list,
+            therefore ``len(output_tensor_lists[i])``) need to be the same
```

Comment:
The formatting was much clearer before this patch, did it render incorrectly?

Reply:
No, it didn't render correctly, that's why I have to change the format.  Basically, we can't have new lines for each of the arguments